Rules/PSScriptAnalyzer.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>PSScriptAnalyzer.Library</id>
+    <version>1.18.3</version>
```

Comment:
You've bumped the version to 1.18.4 in other places. Generally speaking, we should not need a nuspec and just set the equivalent properties in the csproj but if you already have it working, I don't mind.

Reply:
thanks - I followed the same pattern I used for the powershell standard library.